sources/lib/api/gcp/stackdriver/monitoring/_impl/http.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Implements HTTP client helper functionality.
```

Comment:
I don't understand why this exists. If the plan is to be independent of Datalab then why not decouple entirely and we can just include this package as a dependency in gcp.datalab to support the magics? And if not, then this should go away.